includes/wikia/services/HubService.class.php

```diff
@@ -61,6 +61,41 @@ public static function getCurrentWikiaVerticalName() {
 	}
 
 	/**
+	 * Get canonical vertical name for given cityId.
+	 * For Lifestyle and Gaming return their names.
+	 * For rest of values return Entertainment.
+	 * @param $cityId
+	 * @return Boolean|String
+	 */
+	public static function getVerticalNameForComscore( $cityId ) {
+		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $cityId );
+
+		switch ($verticalId) {
+			case WikiFactoryHub::HUB_ID_VIDEO_GAMES:
+				return 'gaming';
+			case WikiFactoryHub::HUB_ID_LIFESTYLE:
+				return 'lifestyle';
+			case WikiFactoryHub::HUB_ID_OTHER:
+				if ( self::isWikiCorporate( $cityId ) ) {
+					return false;
+				} else {
+					return 'lifestyle';
+				}
+			default:
+				return 'entertainment';
+		}
+	}
+
+	private static function isWikiCorporate( $cityId ) {
```

Comment:
I agree with Damian. This function is not simply returning vertical for given wikia but applies some mapping specific for Comscore.

I renamed HUB to VERTICAL and used WikiaPageType method.

ping @kvas-damian  @owend